app/src/main/java/org/mozilla/focus/widget/AnimatedProgressBar.kt

```diff
@@ -0,0 +1,195 @@
+/* -*- Mode: Java; c-basic-offset: 4; tab-width: 4; indent-tabs-mode: nil; -*-
```

Comment:
Please remove this mode line. It's a relic from Fennec and we try to get rid of it whenever we can :)